test/data/ClassType/ClassType.ili

```diff
@@ -0,0 +1,54 @@
+INTERLIS 2.3;
+
+/** KANTONALES DATENMODELL DER KOMMUNALEN NUTZUNGSNPLANUNG GeoIV ID 73 - 
+    Das Modell stellt eine Erweiterung des MGDM des ARE dar.
+    Die Erweiterung besteht in der UUIDOID-Deklaration fuer alle Datenthemen des Modells.
+    2018-03-06: Aenderung zum Import auf Nutzungsplanung_LV95_V1_1.
+    2018-06-22: Erweiterung Assoziation Typ-Typ_Kt. Korrektur TransferMetadaten.
+ */
+!!@ technicalContact=geoinformation@gl.ch
+!!@ IDGeoIV=73
+MODEL GL_Nutzungsplanung_V1_3 (de) AT "http://geo.gl.ch" VERSION "2018-06-22"  =

```

Comment:
Do no use existing MDMs (that depend on other models). Create a simple test model that contains only the model elements that your test requires.